README.md

```diff
@@ -14,6 +14,20 @@ Although this project is very new, it has been thoroughly tested in all modern b
 
 __[Demo](http://jakesidsmith.github.io/react-reorderable/)__
 
+## Installation
+
+* Using npm
+  ```
+  npm install react-reorder
+  ```
+    Add `--save` or `-S` to update your package.json
+
+* Using bower
+  ```
+  bower install react-reorderable
```

Comment:
As in, leave the old name on bower, copy the repo to the new name, add the new name to bower, update NPM and the old repo to point to the new repo?